### PR TITLE
Download button fix

### LINKED ISSUE #88: ui: redesign implementation: download button not seen
Download button is not showing up on page so need to fix that.
Also, have download icon has the button instead of the word "Download"

Does this have something to do with issue #86?

### PR DESCRIPTION
**Pull request by akshara-sun**
To-Do:

- [x] Style and format the download button 

Fixes #88

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -9,10 +9,10 @@
     --secondary-color:#FBDC57;
   }
 }
-
+/*
```
Comment (amilajack) [inline src/styles.css:12]: Why comment these?

File: src/styles.css [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
+@tailwind utilities; */
```
Comment (amilajack) [inline src/styles.css:15]: Same question here

File: src/styles.css [hunk 3]
```diff
@@ -31,10 +31,9 @@ h1, h1, h3, h4, h5, h6 {
   text-align: center;
 }
 
-.bg-default {
+/* .bg-default {
```
Comment (amilajack) [inline src/styles.css:34]: ditto

**Review by amilajack — COMMENTED** [review]
This is great! Good work so far 😃

File: src/styles.css (hunk 1 above)
Comment (amilajack) [reply src/styles.css:12]: @akshara-sun

Comment (akshara-sun) [reply src/styles.css:12]: Will fix these!

**Comment by amilajack** [comment]
Same comment from https://github.com/amilajack/freesound-player/pull/93#issuecomment-705964735 applies here as well

File: src/styles.css (hunk 1 above)
Comment (akshara-sun) [reply src/styles.css:12]: @amilajack I clicked on the "Resolve Conflicts" and took out the comments and "Marked As Resolved"

Comment (akshara-sun) [reply src/styles.css:12]: Not sure what I need to do in Sound.js to fix the merge conflicts

**Review by amilajack — APPROVED** [review]

**Pull request merged**